### DISCUSSION TITLE
Brython support icons for forum etc...

### DISCUSSION THREAD

**Discussion by BobSquarePants** [brython-dev/brython#2328] [General]
Hi I've created two icon / userbar to display wherever you want to support Brython.

The question is I don't where to post them ?

Just here

`mat` 
![Brython_UserBar_Mat](https://github.com/brython-dev/brython/assets/23012674/24f6fa7c-d24c-4a04-9c1c-833c33bb4e63)

`Glossy`
![Brython_UserBar_Glossy](https://github.com/brython-dev/brython/assets/23012674/24c08298-45e4-4a7a-8e7e-ce0b915646de)

or make also new page in the wiki ?

Thanks.

**Comment by denis-migdal** [comment]
Shouldn't you add something like "This Website is powered by Brython" or something like that ?

**Reply by BobSquarePants** [reply]
@denis-migdal , this was for user forum signature, but indeed I will make one to display on webpage :)

**Reply by BobSquarePants** [reply]
I'm looking which size I'll pick to do this Brython support image.

Any ideas or request ?
